ArcGISRuntimeSDKQt_CppSamples/Scenes/GetElevationAtPoint/GetElevationAtPoint.cpp

```diff
@@ -0,0 +1,134 @@
+// [WriteFile Name=GetElevationAtPoint, Category=Analysis]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "GetElevationAtPoint.h"
+
+#include "ArcGISTiledElevationSource.h"
+#include "GraphicsOverlay.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "SimpleMarkerSymbol.h"
+#include "Surface.h"
+
+using namespace Esri::ArcGISRuntime;
+
+GetElevationAtPoint::GetElevationAtPoint(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_scene(new Scene(Basemap::imagery(this), this))
+{
+  // create a new elevation source from Terrain3D REST service
+  ArcGISTiledElevationSource* elevationSource = new ArcGISTiledElevationSource(
+        QUrl("https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"), this);
+
+  // add the elevation source to the scene to display elevation
+  m_scene->baseSurface()->elevationSources()->append(elevationSource);
+
+  // create a graphics overlay to display the elevation marker
+  m_graphicsOverlay = new GraphicsOverlay(this);
```

Comment:
this can go in the initializer list

Reply:
Moved graphic and symbol construction into the initializer list as discussed